workers/ohsome_quality_analyst/cli_opts.py

```diff
@@ -18,6 +18,8 @@ class IntOrStrParamType(click.ParamType):
     https://click.palletsprojects.com/en/8.0.x/parameters/#implementing-custom-types
     """
 
+    name = "interger|string"
```

Comment:
I think it should be `integer|string`  not inte**R**ger

Reply:
Is fixed. I need your approval again to be able to merge this PR.